tools/downloader/caffe2_to_onnx.py

```diff
@@ -0,0 +1,92 @@
+import argparse
+from pathlib import Path
+import sys
+import os
+import re
+
+import onnx
+from caffe2.python.onnx.frontend import Caffe2Frontend
+from caffe2.proto import caffe2_pb2
+
+
+def positive_int_arg(values):
+    """Check positive integer type for input argument"""
+    result = []
+    shapes = re.findall(r'[(\[]([0-9, -]+)[)\]]', values)
+    for shape in shapes:
+        single_shape = []
+        for value in shape.split(','):
+            try:
+                ivalue = int(value)
+                if ivalue < 0:
+                    raise argparse.ArgumentTypeError('Argument must be a positive integer')
+                single_shape.append(ivalue)
+            except Exception as exc:
+                print(exc)
+                sys.exit('Invalid value for input argument: {!r}, a positive integer is expected'.format(value))
+        result.append(single_shape)
+    return result
+
+def parse_args():
+    """Parse input arguments"""
+
+    parser = argparse.ArgumentParser(description='Conversion of pretrained models from Caffe2 to ONNX')
+
+    parser.add_argument('--model-name', type=str, required=True,
+                        help='Model to convert. May be class name or name of constructor function')
```

Comment:
The help message for this is clearly wrong. Also, I have to ask: is it actually necessary to set the model name? What'll happen if we don't?

Reply:
The `onnx.checker` wont pass if the `net.name` is not set.